gensrc/thrift/AgentService.thrift

```diff
@@ -41,6 +41,15 @@ struct TTabletSchema {
     6: optional double bloom_filter_fpp
 }
 
+// this enum stands for different storage format in src_backends
+// V1 for Segment-V1
+// V2 for Segment-V2
+enum TStorageFormat {
+    DEFAULT,
```

Comment:
DEFAULT == V1?

Reply:
It is decided by situation, maybe change from V1 to V2.